doc/content/verification_and_validation/val-2c.md

```diff
@@ -114,34 +113,117 @@ The case and model parameters used in both approaches in TMAP8 are listed in [va
+| --------------- | ----------------------------------------------------------- | ----------------------- | ---------- |
+| $K^0$           | $\boldsymbol{2.8} \times 2.0 \times 10^{-10}$               | 2.823 $\times 10^{-11}$ | m$^3$/Ci/s |
+| $D^e$           | 4.0 $\times 10^{-12}$                                       | 3.155 $\times 10^{-12}$ | m$^2$/s    |
+| $D^w$           | 1.0 $\times 10^{-14}$                                       | 1.846 $\times 10^{-14}$ | m$^2$/s    |
+| $K_S^e$         | $\boldsymbol{1.0 \times 10^{-3}} \times 4.0 \times 10^{19}$ | 5.779 $\times 10^{15}$  | 1/m$^3$/Pa |
+| $K_S^w$         | $\boldsymbol{3.0 \times 10^{-4}} \times 6.0 \times 10^{24}$ | 9.496 $\times 10^{20}$  | 1/m$^3$/Pa |
+| $t_{injection}$ | 10800                                                       | 9838                    | s          |
+
+## Results and Discussion
 
 [val-2c_comparison_T2] and [val-2c_comparison_HTO] show the comparison of the TMAP8 calculations (both with immediately injected and delayed injected T$_2$) against the experimental data for T$_2$ and HTO concentration in the enclosure over time.
 There is reasonable agreement between the TMAP8 predictions and the experimental data.
-In the case of immediate T$_2$ injection,  the root mean square percentage errors are equal to RMSPE = 58.98 % for T$_2$ and RMSPE = 139.10 % for HTO, respectively.
+In the case of immediate T$_2$ injection, the root mean square percentage errors (RMSPE) are equal to RMSPE = 58.98 % for T$_2$ and RMSPE = 139.10 % for HTO, respectively.
 When accounting for a delay in T$_2$ injection, the TMAP8 predictions best match the experimental data, in particular the position of the peak HTO concentration. The RMSPE values decrease to RMSPE = 58.05 % for T$_2$ and RMSPE = 74.77 % for HTO, respectively.
 Note that the model parameters listed in [val-2c_parameters] are somewhat different from [!cite](Holland1986,longhurst1992verification,ambrosek2008verification) to better match the experimental data.
-In particular, [!cite](longhurst1992verification,ambrosek2008verification) did not validate the TMAP predictions against T$_2$ concentration, which we do here in [val-2c_comparison_T2].
+In particular, [!cite](longhurst1992verification,ambrosek2008verification) did not validate the TMAP predictions against T$_2$ concentration, which we do here in [val-2c_comparison_T2] and in [!cite](Simon2025).
 This affects some of the model parameters.
 
-The agreement between modeling predictions and experimental data could be further improved by adjusting the model parameters.
-It is also possible to perform this optimization with [MOOSE's stochastic tools module](https://mooseframework.inl.gov/modules/stochastic_tools/index.html).
-
 !media comparison_val-2c.py
        image_name=val-2c_comparison_TMAP8_Exp_T2_Ci.png
        style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
```

Comment:
Leaving a single comment here to highlight an issue I am seeing with several figures. I think the width needs to be increased somewhat in order to make it more readable on smaller displays. Yes, I can click on it to make it bigger, but at a glance the width sacrifices some of the "casual" readability of the page. Instead of 50%, we should consider something closer to 70%, I think.